### ISSUE TITLE
Add a class to mentions to differentiate them from all other toots

### ISSUE THREAD

**Issue by rknightuk** [mastodon/mastodon#8424]
tl;dr: Add a class to mentions, like DMs have (`status-direct`) for targeting with custom CSS.

****

One of the customisations I'm doing in Mastodon when running in [Fluid](https://fluidapp.com/) is changing the colour of DMs and mentions. DMs have `status-direct`, boosts have `notification-favourite` and so on. Without a specific class on mentions (they have a generic `status-public`), it's impossible to highlight mentions in the feed and the notifications column (I can target the notifications column, but only using `nth-child` which doesn't work if the columns are rearranged).

This also becomes more valuable with #8399 as instance admins will be able to add custom CSS.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by ClearlyClaire** [comment]
Hi! Is this still something you want/need? If so, can you reply to https://github.com/mastodon/mastodon/pull/30080#issuecomment-2093132092 explaining your use case?

**Comment by rknightuk** [comment]
No I don't need this now, I'll close the issue.
